src/Compiler/Microsoft.AspNetCore.Mvc.Razor.Extensions.Version2_X/test/TestFiles/IntegrationTests/CodeGenerationIntegrationTest/Basic_Runtime.ir.txt

```diff
@@ -3,13 +3,13 @@
     CSharpCode - 
         IntermediateToken -  - CSharp - [assembly:global::Microsoft.AspNetCore.Mvc.Razor.Compilation.RazorViewAttribute(@"/TestFiles/IntegrationTests/CodeGenerationIntegrationTest/Basic.cshtml", typeof(AspNetCore.TestFiles_IntegrationTests_CodeGenerationIntegrationTest_Basic))]
     NamespaceDeclaration -  - AspNetCore
-        UsingDirective - (1:0,1 [14] ) - System
-        UsingDirective - (16:1,1 [34] ) - System.Collections.Generic
-        UsingDirective - (51:2,1 [19] ) - System.Linq
-        UsingDirective - (71:3,1 [30] ) - System.Threading.Tasks
-        UsingDirective - (102:4,1 [32] ) - Microsoft.AspNetCore.Mvc
-        UsingDirective - (135:5,1 [42] ) - Microsoft.AspNetCore.Mvc.Rendering
-        UsingDirective - (178:6,1 [45] ) - Microsoft.AspNetCore.Mvc.ViewFeatures
+        UsingDirective - (1:0,1 [12] ) - System
```

Comment:
Every using directive is now 2 character shorter to account as it doesnt include the `\r\n`